unified_planning/test/test_pddl_io.py

```diff
@@ -305,44 +305,26 @@ def test_examples_io(self):
                 w.write_problem(problem_filename)
 
                 reader = PDDLReader()
+                if problem.name != 'hierarchical_blocks_world_with_object':
```

Comment:
What is this? I think it's better to look in the ProblemKind to skip the non-action-based tests, no?